lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1022,6 +1022,14 @@ def _create_deployment_parameters(
         ):
             arm_parameters.use_availability_sets = True
 
+        if environment.platform and (
+            "Infiniband"
+            in [feature.name() for feature in environment.platform.supported_features()]
```

Comment:
The logic is not useful. The code is in Azure platform, it supports Infiniband as we know. It needs to check, if the Infiniband is supported by the capability of one of node. If so, it needs below settings.